user/src/helper/axios.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { api } from "../urlConfig";
+import store from "../store";
+import { authConstant } from "../actions/constant";
+
+const token = window.localStorage.getItem("token");
+console.log(token);
+
+const axiosInstance = axios.create({
+    baseURL: api,
+    headers: {
+        Authorization: token ? token : "",
```

Comment:
```suggestion
        ...(token && { Authorization: token })
```
Nếu không có token thì sẽ không có truyền Authorization luôn nhé